packages/phone-number-privacy/combiner/src/bls/bls-cryptography-client.ts

```diff
@@ -1,5 +1,5 @@
 import threshold_bls from 'blind-threshold-bls'
-import { ErrorMessages } from '../common/error-utils'
+import { ErrorMessage } from '../common/error-utils'
```

Comment:
Sorry for the noise from this rename. It's a more common convention in Typescript to give enums singular names because when they're used as types, what you're passing in is *one* ErrorMessage and not *many* ErrorMessages